api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -150,8 +156,13 @@ public void testSyncEraCommonsStatus() throws Exception {
     assertThat(user.getEraCommonsCompletionTime()).isEqualTo(
         new Timestamp(TIMESTAMP_MSECS));
     assertThat(user.getEraCommonsLinkExpireTime()).isEqualTo(
-        new Timestamp(TIMESTAMP_MSECS + 1));
+        new Timestamp(TIMESTAMP_MSECS / 1000));
     assertThat(user.getEraCommonsLinkedNihUsername()).isEqualTo("nih-user");
+
+    // Completion timestamp should not change when the method is called again.
+    Timestamp completionTime = user.getEraCommonsCompletionTime();
+    userService.syncEraCommonsStatus();
+    assertThat(user.getEraCommonsCompletionTime()).isEqualTo(completionTime);
```

Comment:
what will happen if this assertion fails?

Reply:
This was meant to be a regression test for the logic change to avoid updating the completion time , following the pattern we use for 2fa at L196.

Looking at this now, I realize that the current timestamp isn't changing between calls, so it wouldn't actually catch a regression. Fixed.